packages/inventory/src/Stocktake/DetailView/columns.ts

```diff
@@ -8,40 +8,47 @@ import {
   Column,
   SortBy,
 } from '@openmsupply-client/common';
-import { StocktakeLine, StocktakeSummaryItem } from './../../types';
+import { StocktakeSummaryItem } from './../../types';
```

Comment:
this is not ideal and might be a reason why having the flattened item fields could be a better way to go. Alternatively, making this function be able to handle nested paths. For now, I've hacked away in favour of getting your opinion/doing a proper fix in a dedicated issue

Reply:
I was wondering about extracting an array of items and using that
```
              const items = getItems(row);
              return ifTheSameElseDefault(items, 'code', '');
```

with 
```
const getItems = (row: StocktakeSummaryItem) => {
  const { lines } = row;
  const items =
    lines?.filter(line => !!line.item).map(line => line.item as ItemNode) ?? [];
  return items;
};
```

similar to supporting nested paths - just making it explicit at this level 🤷 

I quite like the `item` object.. but it is much more annoying to actually use

Reply:
Ah yes, thanks, that's a very nice approach.